modules/gcp-gke/main.tf

```diff
@@ -90,6 +90,17 @@ resource "google_container_cluster" "primary" {
     }
   }
 
+  dynamic "maintenance_policy" {
+    for_each = var.maintenance_policy_config != null ? var.maintenance_policy_config : []
```

Comment:
maintenance_policy_config default value is empty. So need of validating with !=null